common/components/Pages/Profile/index.tsx

```diff
@@ -108,21 +111,17 @@ export const ProfilePage: React.FC = () => {
             )}
           />
         </Card>
-        {/* TODO: Profile edit form */}
         <Card title="Інформація користувача" style={{ flex: 1 }}>
-          {/* <Spin>Можливість редагування профілю вже в процесі розробки ^_^</Spin> */}
-          <AddProfileEditForm />
-          {/* <Form>
-            <Typography.Text editable>description</Typography.Text>
-            <Divider />
-          </Form> */}
+          <EditUserForm userId={`${user?._id}`} form={form}/>
```

Comment:
why ``userId={`${user?._id}`}``? why not just `userId={user?._id}`? It it is type conflict - use `.toString()`